src/course-unit/hooks.jsx

```diff
@@ -121,6 +128,18 @@ export const useCourseUnit = ({ courseId, blockId }) => {
     setXBlocksExpanded((prevState) => !prevState);
   };
 
+  const handleRollbackMovedXBlock = () => {
+    dispatch(rollbackUnitItemQuery(blockId, movedXBlockParams.sourceLocator, movedXBlockParams.title));
+  };
+
+  const handleCloseXBlockMovedAlert = () => {
+    dispatch(updateMovedXBlockParams({ isSuccess: false }));
+  };
+
+  const handleNavigateToTargetUnit = () => {
+    navigate(`/course/${courseId}/container/${movedXBlockParams.targetParentLocator}`);
+  };
```

Comment:
[question] Does this navigation work correct with `basename` like `/course-authoring/`?
Or we need to use the utils function `createCorrectInternalRoute` to normalize the internal path?

Reply:
Yeah, replaced
Good catch 💯